rental.rb

```diff
@@ -0,0 +1,26 @@
+class Rental
+  attr_reader :student, :book, :person
```

Comment:
- [ ] Let's kindly make sure that a book and a person should be modified(`attr_accessor :date, :book, :person`) 👍